recipes/Java/JodaTimeToJavaTime/DateTimeFormatterBuilder/datetimeformatterbuilder-appendTimeZoneShortName-map.yaml

```
@@ -0,0 +1,27 @@
+id: scw:java.time:Joda-Time:datetimeformatterbuilder-appendTimeZoneShortName-map
+version: 10
+metadata:
+  name: Rewrite appendTimeZoneShortName to appendZoneText
+  shortDescription: Rewrite appendTimeZoneShortName to appendZoneText
+  level: warning
+  language: java
+  enabled: true
+  comment: |
+    Searches for a append* method called and provides fixes to migrate to a java.time equivalent.
+    This recipe is designed to match on broken code. This method originally was called by joda DateTimeFormatterBuilder type,
+    however this object type should first be migrated to java.time.format.DateTimeFormatterBuilder using the other migration recipes.
+    This recipe will then match on the broken code, and the fixes in this recipe will allow the completion of the migration.
+  descriptionFile: descriptions/datetimeformatterbuilder.html
+  tags: java.time;framework specific;Joda-Time;quality
+search:
+  methodcall:
+    argCount: 1
+    name: appendTimeZoneShortName
+    type: java.time.format.DateTimeFormatterBuilder
+availableFixes:
+- name: Rewrite using appendZoneText
+  actions:
+  - rewrite:
+      to: '{{{ qualifier }}}.appendZoneText(TextStyle.SHORT, {{{ arguments }}})'
```

TextStyle needs to be fully qualified
```suggestion
      to: '{{{ qualifier }}}.appendZoneText(java.time.format.TextStyle.SHORT, {{{ arguments }}})'
```